test/libsolidity/semanticTests/constants/constant_string.sol

```diff
@@ -16,7 +16,9 @@ contract C {
     }
 }
 
+// ====
+// compileViaYul: also
 // ----
-// f() -> 0x20, 3, "\x03\x01\x02"
-// g() -> 0x20, 3, "\x03\x01\x02"
+// f() -> 0x20, 3, ""
+// g() -> 0x20, 3, ""
```

Comment:
Seems to be broken.

Reply:
hm, that seems to have been just the tool

Reply:
@mijovic can you check this in the tool and perhaps create a tracking issue (to be fixed later)?

Reply:
I'll check behavior of isoltest here

Reply:
I can't reproduce this

Reply:
After rebasing I got it. Will investigate and create tracking issue.

Reply:
#8885